### DISCUSSION TITLE
Migration to another server & change port grafana

### DISCUSSION THREAD

**Discussion by igloox-jee** [teslamate-org/teslamate#3663] [Q&A] [answered]
Hello,
I am looking to migrate my TeslaMate container and its data, while also changing the port of the Grafana container. I have successfully completed the backup and import on the new server. I can access TeslaMate via port 4000, but when I try to access a dashboard, the link redirects me to the IP and port of the old server. Is there a way to modify this in a configuration file?
Also, Is there a way to change the Grafana port and ip in app while importing the TeslaMate database from my old server?
Thank you in advance for your assistance.

**Comment by cwanja** [comment, marked as answer]
Update the URLs in the **Settings** page.
